docs/source/reference/artifacts.rst

```diff
@@ -5,12 +5,16 @@ optuna.artifacts
 
 The :mod:`~optuna.artifacts` module provides the way to manage artifacts (output files) in Optuna.
 
-.. autosummary::
```

Comment:
I use `autoclass` and `autofunction` directives since`autosummary` seems not to have the option to make `Methods` section hidden.
https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html